python/modules/DeepTopProducer.py

```diff
@@ -295,23 +297,19 @@ def calculateResTopSFWeight(self, res):
         resTopGM = np.array([topCand.genMatch for topCand in res], dtype=bool)[resTopStop0l]
         resTopDisc = np.array([topCand.discriminator for topCand in res], dtype=float)[resTopStop0l]
 
+        discCut = resTopDisc > self.DeepResolveWP
```

Comment:
@pastika I thought I merge from master. But seems we have conflict. Can you confirm which code I should get?

Reply:
this moved from below, but I think it can go here or where it was in master, I ended up not needing it here after all